### PR TITLE
Find a keyboard device and use that for evemu

### PR DESCRIPTION
**Pull request by FossPrime**
My mouse is not keyboard capable and wont work. This seems like the most reliable way to find a keyboard and emit events from that.

### CODE REVIEW

File: mousemapper.sh [hunk 1]
```diff
@@ -4,16 +4,17 @@ event_type=EV_KEY
 action_type=POINTER_BUTTON
 pressed="pressed,"
 
-readarray -t devices <<<$(libinput list-devices | grep pointer -B3 | grep -o '/dev/input/event[1-9]*')
+readarray -t devices <<<$(libinput list-devices | grep pointer -B3 | grep -o '/dev/input/event[0-9]*')
+keyboard=$(ls /dev/input/by-path/*-kbd)
```
Comment (ah-) [inline mousemapper.sh]: Won't this break if you have two keyboards?

Comment (FossPrime) [reply mousemapper.sh]: fixed

**Comment by FossPrime** [comment]
Had another probably unrelated issue... mouse-mapper stopped triggering on button presses. I can't tell if that's because the device number changed, or because of my xmodmap disabling of the buttons. Will add a better message to figure it out.

**Comment by FossPrime** [comment]
It turns out disconnecting the mouse will remove /dev/input/eventX and break libinput debug. Will submit my patch for that.

**Comment by FossPrime** [comment]
Found a way to disable annoying btn_extra handlers like Nautilus... which don't respond to X based approaches. This is far simpler and more stable than the high level libinput approach.: https://unix.stackexchange.com/a/516173/38213

**Comment by FossPrime** [comment]
Please let me know fi you need anything else to merge

**Comment by FossPrime** [comment]
libinput debug is not printing keyboard events anymore :(
I have 1.14.2 and I'm using bluetooth

> libinput debug-events tool creates a libinput context and prints all events from these devices. This is a debugging tool only, its output may change at any time. Do not rely on the output. 

> libinput bug: Event for missing capability CAP_KEYBOARD on device "MX Anywhere 2S Mouse"
 event23  POINTER_MOTION    +5.44s	  0.00/ -0.88 ( +0.00/ -1.00)
libinput bug: Event for missing capability CAP_KEYBOARD on device "MX Anywhere 2S Mouse"

**Comment by mathportillo** [comment]
There is a new update using lib-input to list-devices!

**Comment by mathportillo** [comment]
> Found a way to disable annoying btn_extra handlers like Nautilus... which don't respond to X based approaches. This is far simpler and more stable than the high level libinput approach.: https://unix.stackexchange.com/a/516173/38213

sorry for taking so long to answer, but indeed this is made for non X versions. X is much easier to deal.

**Pull request closed without merging**